change/@fluentui-tokens-95ae3ced-8ddb-464e-8f6c-ffcce322902c.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
## 🕵🏾‍♀️ [<b data-uid='vrt-gist'></b> visual regressions to review in the fluentuiv9 Visual Regression Report](https://aka.ms/visual-changes-review?organization=uifabric&project=fabricpublic&prId=33064&commitId=b1ad8e411cfb6035c918643f736fa70aaa7a7078&env=prod&clientType=FLUENTUI)


  <details><summary><b>Avatar Converged</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Avatar Converged.Badge Mask RTL.chromium.png|3| Changed|
|Avatar Converged.badgeMask.chromium.png|3| Changed|

</details>


  <details><summary><b>Drawer</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Drawer.Full Overlay High Contrast.chromium.png|2231| Changed|
|Drawer.Full Overlay RTL.chromium.png|3279| Changed|

</details>


  <details><summary><b>Menu Converged - selection</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Menu Converged - selection.Switch High Contrast.chromium.png|157| Changed|

</details>


  <details><summary><b>SwatchPicker Converged</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|SwatchPicker Converged.Default High Contrast.chromium.png|174| Changed|

</details>


  <details><summary><b>Switch Converged</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Switch Converged.Enabled And Checked High Contrast.chromium.png|157| Changed|

</details>


  <details><summary><b>Table layout flex</b> 4 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Table layout flex.Multiselect Checked Neutral High Contrast.chromium.png|76| Changed|
|Table layout flex.Multiselect Checked Brand High Contrast.chromium.png|76| Changed|
|Table layout flex.Multiselect Mixed Neutral High Contrast.chromium.png|30| Changed|
|Table layout flex.Multiselect Mixed Brand High Contrast.chromium.png|30| Changed|

</details>


  <details><summary><b>Table layout table</b> 4 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Table layout table.Multiselect Checked Brand High Contrast.chromium.png|76| Changed|
|Table layout table.Multiselect Checked Neutral High Contrast.chromium.png|76| Changed|
|Table layout table.Multiselect Mixed Brand High Contrast.chromium.png|30| Changed|
|Table layout table.Multiselect Mixed Neutral High Contrast.chromium.png|30| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>